lib/ibed.ml

```diff
@@ -0,0 +1,194 @@
+type item = {
+    bait_names: string list;
+    bait: GLoc.t;
+    other_names: string list;
+    other: GLoc.t;
+    n_reads: int;
+    score: float;
+  }
+
+type t = item list
+
+let head = "bait_chr\tbait_start\tbait_end\tbait_name\totherEnd_chr\totherEnd_start\totherEnd_end\totherEnd_name\tN_reads\tscore"
+
+let ibed_fmt = ("%s\t%d\t%d\t%s\t%s\t%d\t%d\t%s\t%d\t%f" : _ format)
+
+let ibed_fmt6 = ("%s\t%d\t%d\t%s\t%s\t%d\t%d\t%s\t%d\t%f" : _ format6)
+
+let order_baits it1 it2 =
```

Comment:
I'd prefer `order_by_bait` since it's really items that are ordered by this function, according to their bait.